### PR TITLE
Updated remaining VPN logos to use Protocol assets

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
Some pages were using VPN logos not from Protocol, so I updated that in the necessary templates. 

**Note:** The PR number here is a very nice angel number ✨👼🏼 this means good vibes 🏄🏻‍♀️

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/10243
## Testing
http://localhost:8000/en-US/products/vpn/resource-center/
http://localhost:8000/en-US/privacy/mozilla-vpn/
http://localhost:8000/en-US/products/vpn/

### CODE REVIEW

File: media/css/products/vpn/resource-center.scss [hunk 1]
```diff
@@ -50,12 +50,7 @@ $image-path: '/media/protocol/img';
 }
 
 .resource-center-wordmark {
-    @include image-replaced;
```
Comment (reemhamz) [inline media/css/products/vpn/resource-center.scss:53]: Removed VPN as a background image here and just placed it as a class in the HTML page.

File: bedrock/products/templates/products/vpn/resource-center/landing.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
     block_class='resource-center-split mzp-t-content-lg mzp-l-split-center-on-sm-md',
     media_class='resource-center-split-media',
   ) %}
-    <h1 class="resource-center-wordmark">{{ ftl('vpn-resource-center-mozilla-vpn') }}</h1>
+    <h1 class="resource-center-wordmark mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-vpn">{{ ftl('vpn-resource-center-mozilla-vpn') }}</h1>
```
Comment (reemhamz) [inline bedrock/products/templates/products/vpn/resource-center/landing.html:52]: Added the VPN logo for this page using classes

File: media/css/products/vpn/resource-center.scss (hunk 1 above)
Comment (maureenlholland) [reply media/css/products/vpn/resource-center.scss:53]: This is an open issue with Protocol wordmarks and @craigcook details the difficulty of resolving it here: https://github.com/mozilla/protocol/issues/718

**suggestion (blocking):** Add a new media query for mobile screens only `@media screen and (max-width: #{$screen-md}) ` and set `background-position: top center` on the wordmark here. Also worth adding a comment on these style rules to make clear it's a patch fix (similar to https://github.com/mozilla/bedrock/pull/10874/files#diff-31b556a34950073624d0165cb5376477f1ef2eaf575f0d88629abfa776b2df60R13). You don't have to use this solution exactly, but some patch fix to fully center the workmark on mobile is required.

File: bedrock/products/templates/products/vpn/resource-center/landing.html (hunk 2 above)
Comment (maureenlholland) [inline bedrock/products/templates/products/vpn/resource-center/landing.html:52]:
```suggestion
    <h1 class="resource-center-wordmark mzp-c-wordmark mzp-t-wordmark-sm mzp-t-product-vpn">{{ ftl('vpn-resource-center-mozilla-vpn') }}</h1>
```

**suggestion:** reducing the wordmark size will match production better

File: bedrock/privacy/templates/privacy/notices/mozilla-vpn.html [hunk 3]
```diff
@@ -8,4 +8,4 @@
 
 {% set body_id = "mozilla-vpn" %}
 
-{% block article_header_logo %}{{ static('img/logos/vpn/logo-with-wordmark.svg') }}{% endblock %}
```
Comment (maureenlholland) [inline bedrock/privacy/templates/privacy/notices/mozilla-vpn.html:11]: **suggestion (blocking)**: let's delete this outdated `img/logos/vpn/logo-with-wordmark.svg` file. I don't see any other instances of use now, you caught them all 🥅

**Review by maureenlholland — CHANGES_REQUESTED** [review]
Thanks for picking this up! Tricky case on the VPN resource page, but I think a patch fix is acceptable. Really happy to see the consistency of these VPN Protocol assets!! 🎉

File: media/css/products/vpn/resource-center.scss (hunk 1 above)
Comment (reemhamz) [reply media/css/products/vpn/resource-center.scss:53]: Great way to fix this!

**Review by maureenlholland — APPROVED** [review]
r+ 🧹

**Pull request merged**